nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Js add tag to post

# Description
Feature enabling users to associate and unassociate multiple tags on a post.
Fixes #17 and #18 
## Type of change
# Testing Instructions
- Pull down PR branch
- Make sure sql express and appsettings local file are updated
- Launch API and App
- Navigate to post details
- Click Manage Tags button, will be re-directed to list of tags page
- Select any tags you want associated or unassociated with a post
- Click Save button, database should reflect the addition/removal of PostTag records
- User is re-directed back to the post details view and the associated tags should be displayed
# Checklist:
